data/db-schema/src/main/resources/net/corda/db/schema/statemanager/db.changelog-master.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+    <include file="net/corda/db/schema/statemanager/migration/state-manager-migration-v1.0.xml"/>
```

Comment:
```suggestion
    <include file="net/corda/db/schema/statemanager/migration/state-manager-migration-v5.1.xml"/>
```

I think there was an agreement that the version would match the product revision (5.1, 5.2 etc).
I know this isn't strictly necessary as this is just the first revision of it, but it was argued that it makes it easier to evaluate what's new/changed per version.

Reply:
I assume this is true going forward for all new schema versions i.e. flow config will go to 5.1 in [this ](https://github.com/corda/corda-api/pull/1236/files#diff-70c498ea23446b0bea54e380ff6176087c65615fffa7c38c79a431730f715692)change?